packages/lit/rollup.config.js

```diff
@@ -53,10 +53,27 @@ export default litProdConfig({
     {
       file: 'polyfill-support',
     },
```

Comment:
I noticed this earlier today and thought it might have implications for the naming scheme used below since the polyfill support module is in a UMD wrapper. However, I think this is fine because it actually works in both ESM and UMD contexts as it has no exports (it only writes globals).